include/mbedtls/ssl.h

```diff
@@ -1227,8 +1228,28 @@ struct mbedtls_ssl_session {
     uint32_t MBEDTLS_PRIVATE(ticket_lifetime);   /*!< ticket lifetime hint    */
 #endif /* MBEDTLS_SSL_SESSION_TICKETS && MBEDTLS_SSL_CLI_C */
 
+#if defined(MBEDTLS_SSL_SESSION_TICKETS) && defined(MBEDTLS_SSL_SRV_C) && \
+    defined(MBEDTLS_HAVE_TIME)
+    /*! When a ticket is created by a TLS server as part of an established TLS
+     *  session, the ticket creation time may need to be saved for the ticket
+     *  module to be able to check the ticket age when the ticket is used.
+     *  That's the purpose of this field.
+     *  Before creating a new ticket, an Mbed TLS server set this field with
+     *  its current time in milliseconds. This time may then be saved in the
+     *  session ticket data by the session ticket writing function and
+     *  recovered by the ticket parsing function later when the ticket is used.
```

Comment:
First, thanks for the review.

>Info: why do it this way, rather than letting the ticket writing function query the current time if it wants?

The main reason is for an implementation of `mbedtls_ssl_ticket_write_t` to rely only on `mbedtls_ssl_session_save` to create the ticket session data and not having to set-up session field by itself (`ticket_creation_time` here). That way a third party implementation of ` mbedtls_ssl_ticket_write_t` handling `session` only as a black box with `mbedtls_ssl_session_save` should continue to work.